src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -144,6 +166,9 @@ public final void refreshItems() {
         listView.setItems(null);
         listView.setItems(getElementsList());
         issueCount = getElementsList().size();
+        closedIssueCount = getClosedIssueSize();
+        openIssueCount = issueCount - closedIssueCount;
+        System.out.println("closed = " + closedIssueCount + " open = " + openIssueCount);
```

Comment:
May be we use this format instead `T (O + C)`. If Total is 0, we can omit the part inside brackets.